Sources/MapboxDirections/MapMatching/MapMatchingResponse.swift

```diff
@@ -1,49 +1,46 @@
 import Foundation
 
 public struct MapMatchingResponse {
-    public var code: String
-    public var routes : [Route]?
-    public var waypoints: [Waypoint]
+    public var code: String?
+    public var message: String?
```

Comment:
`code` and `message` can both be encapsulated in `error`, so we should keep `code` and `message` around as parallel state, especially since all these properties are writable.